src/modules/settings/index.js

```diff
@@ -146,7 +147,20 @@ class SettingsModule {
         $('body').append(panel);
 
         cdn.get('privacy.html').then(data => $('#bttvPrivacy .tse-content').html(data));
-        cdn.get('changelog.html', true).then(data => $('#bttvChangelog .tse-content').html(data));
+
+        const changelogPanel = $('#bttvChangelog .tse-content');
+        api.get('changelog').then(data => {
+            if (data.status === 200) {
```

Comment:
if status was not 200, it would not hit this function.. please read up on promises https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise